src/globus_sdk/services/timer/client.py

```diff
@@ -116,3 +116,38 @@ def delete_job(
         """
         log.info(f"TimerClient.delete_job({job_id})")
         return self.delete(f"/jobs/{job_id}")
+
+    def pause_job(
+        self,
+        job_id: UUIDLike,
+    ) -> response.GlobusHTTPResponse:
+        """
+        ``POST /jobs/<job_id>/pause``
+
+        **Examples**
+
+        >>> timer_client = globus_sdk.TimerClient(...)
+        >>> timer_client.pause_job(job_id)
+        """
+        log.info(f"TimerClient.pause_job({job_id})")
+        return self.post(f"/jobs/{job_id}/pause")
+
+    def resume_job(
+        self,
+        job_id: UUIDLike,
+        *,
+        update_credentials: bool | None = None,
+    ) -> response.GlobusHTTPResponse:
+        """
+        ``POST /jobs/<job_id>/resume``
+
```

Comment:
As above, param docs. Suggested content as a starting point for this:

```suggestion

        :param job_id: The ID of the timer to resume
        :type job_id: str or UUID
        :param update_credentials: When true, replace the credentials for the timer based
            on the credentials for this resume call. This can be used to resolve session
            and consent errors on timers, but it also could introduce session and consent
            errors, depending on the credentials being used to resume.
        :type update_credentials: bool, optional
```